rollup.base.config.js

```diff
@@ -0,0 +1,569 @@
+    var gblTokens = gblNs.split(".");
+    let common = [];
+    let idx = 0;
+    while (brTokens.length > idx && gblTokens.length > idx && brTokens[idx] === gblTokens[idx]) {
+        common.push(brTokens[idx]);
+        idx++;
+    }
+
+    return {
+        common: common,
+        browser: brTokens.slice(idx),
+        gbl: gblTokens.slice(idx)
+    };
+};
+
+const getIntro = (format, theNameSpace, moduleName, theVersion) => {
+    let theIntro = "";
+    if (format === "iife" || format === "umd") {
+        let nsTokens = getCommonNamespace(theNameSpace.browser, theNameSpace.gbl);
+        theIntro += "(function (global, factory) {\n";
+        let prefix = "    ";
+        theIntro += prefix + "var undef = \"undefined\";\n";
+        if (format === "umd") {
+            // UMD supports loading via requirejs and 
+            theIntro += prefix + "typeof exports === \"object\" && typeof module !== undef ? factory(exports) :\n";
+            theIntro += prefix + "typeof define === \"function\" && define.amd ? define([\"exports\"], factory) :\n";
+            theIntro += prefix + "(function(global){\n";
+            prefix += "    ";
+        }
+        // Both IIFE and UMD
+        theIntro += prefix + "var nsKey, key, nm, theExports = {}, modName = \"" + moduleName.replace(/[\."\\\/\-]/g, "_") + "\", msMod=\"__ms$mod__\";\n";
+        theIntro += prefix + "var mods={}, modDetail=mods[modName]={}, ver=\"" + theVersion + "\";\n";
+        let baseNs = "global";
+        if (nsTokens.common.length > 0) {
+            theIntro += getNamespace(prefix, nsTokens.common, "baseNs", baseNs);
+            baseNs = "baseNs";
+        }
+        theIntro += prefix + "// Versioned namespace \"" + theNameSpace.browser + "\"\n";
+        theIntro += getNamespace(prefix, nsTokens.browser, "exportNs", baseNs);
+        theIntro += prefix + "// Global namespace \"" + theNameSpace.gbl + "\"\n";
+        theIntro += getNamespace(prefix, nsTokens.gbl, "destNs", baseNs);
+        theIntro += prefix + "var expNsDetail=(exportNs[msMod]=(exportNs[msMod] || {})), expNameVer=(expNsDetail[\"v\"]=(expNsDetail[\"v\"] || []));\n";
+        theIntro += prefix + "var destNsDetail=(destNs[msMod]=(destNs[msMod] || {})), destNameVer=(destNsDetail[\"v\"]=(destNsDetail[\"v\"] || []));\n";
+        theIntro += prefix + "(destNsDetail[\"o\"]=(destNsDetail[\"o\"] || [])).push(mods);\n";
+        theIntro += prefix + "factory(theExports);\n";
+        theIntro += prefix + "for(var key in theExports) {\n";
+        theIntro += prefix + "    // Always set the imported value into the \"export\" versioned namespace (last-write wins)\n";
+        theIntro += prefix + "    nm=\"x\", exportNs[key]=theExports[key], expNameVer[key]=ver;\n";
+        theIntro += prefix + "    // Overwrite every elements in namespace and record (last-write wins)\n";
+        theIntro += prefix + "    typeof destNs[key]===undef ? (nm=\"n\", destNs[key]=theExports[key]) && (destNameVer[key]=ver)";
```

Comment:
This still looks wrong and will probably be generating a JavaScript error as this is an incomplete `? :` operator usage...

I think it should just be (same as 91, except using the destNs and destNameVer)
`nm=\"n\", destNs[key]=theExports[key],  destNameVer[key]=ver;`